source/Octopus.Tentacle/Configuration/TentacleConfiguration.cs

```diff
@@ -1,4 +1,3 @@
-#nullable enable
```

Comment:
Was this intentional?

Reply:
Ah yeah I think that was rider deeming it unnecessary but I'll undo that change to keep this PR focused.

Reply:
Either way is fine :D